src/test/prediction_service_test.cpp

```diff
@@ -224,12 +288,24 @@ class TestPredict : public ::testing::Test {
         }
 
         response.Clear();
+#pragma GCC diagnostic push
+#pragma GCC diagnostic ignored "-Wall"
+        // turn off all warnings since compiler will complain that it cannot cast KFS/TFS
+        // request types to InferenceRequest
+        //   const ovms::InferenceRequest* irq = dynamic_cast<const ovms::InferenceRequest*>(&request);
+        // ovms::InferenceResponse* irs = dynamic_cast<ovms::InferenceResponse*>(&response);
+#pragma GCC diagnostic pop
+        // if ((irq != nullptr) && (irs != nullptr)) {
```

Comment:
Why do we leave such comments in the code?

Reply:
Leftover to be removed